flax/struct.py

```diff
@@ -30,14 +30,16 @@
 """Utilities for defining custom classes that can be used with jax transformations.
 """
 
+from typing import Type, TypeVar
+
 from . import serialization
 
 import dataclasses
 
 import jax
 
 
-def dataclass(clz):
+def dataclass(clz: Type):
```

Comment:
I think you can also just write `type` instead of `Type`, saving you an import :-)

Reply:
Done